src/components/ManageAccountsDrawer/components/ImportAccounts.tsx

```diff
@@ -333,9 +370,33 @@ export const ImportAccounts = ({ chainId, onClose, isOpen }: ImportAccountsProps
         if (isEnabled) {
           return
         }
-        await dispatch(
-          portfolioApi.endpoints.getAccount.initiate({ accountId, upsertOnFetch: true }),
-        )
+
+        // "Fetch" the query leveraging the existing cached data
```

Comment:
i.e no actual fetch happening here (once again assuming 5mn of opening the app, see comment above)